.evergreen/config.in.yml

```diff
@@ -819,7 +819,9 @@ functions:
           rm -f ./prepare_client_encryption.sh
 
           VERSION=${VERSION} DRIVERS_TOOLS=${DRIVERS_TOOLS} \
-            bash ${PROJECT_DIRECTORY}/.evergreen/prepare-crypt-shared-lib.sh
+            . ${PROJECT_DIRECTORY}/.evergreen/prepare-crypt-shared-lib.sh
+
+          echo "setting CRYPT_SHARED_LIB_PATH: $CRYPT_SHARED_LIB_PATH"
```

Comment:
This echo is here  and the var is definitely set: https://evergreen.mongodb.com/task_log_raw/mongo_node_driver_next_ubuntu1804_custom_dependency_tests_run_custom_csfle_shared_lib_tests_pinned_commit_patch_c70c69edb7b51aeac143aa2eb73aaaf465c41572_62f12690d1fe07159c0476cb_22_08_08_15_06_57/0?type=T#L260